common/rendering.h

```diff
@@ -1084,6 +1084,7 @@ namespace rs2
             case RS2_FORMAT_RGBA8: case RS2_FORMAT_BGRA8: // Display both RGBA and BGRA by interpreting them RGBA, to show the flipped byte ordering. Obviously, GL_BGRA could be used on OpenGL 1.2+
                 glTexImage2D(GL_TEXTURE_2D, 0, GL_RGBA, width, height, 0, GL_RGBA, GL_UNSIGNED_BYTE, data);
                 break;
+            case RS2_FORMAT_CONFIDENCE:
```

Comment:
Is confidence a format or a stream type?

Reply:
RS2_FORMAT_CONFIDENCE is under RS2_STREAM_CONFIDENCE_MAP.
Maybe I should change the format to be RS2_FORMAT_DEPTH_CONFIDENCE